dev-util/flawfinder/flawfinder-2.0.7.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_6 )
+
+inherit distutils-r1
+
+DESCRIPTION="Examines C/C++ source code for security flaws"
+HOMEPAGE="https://www.dwheeler.com/flawfinder/"
+SRC_URI="https://www.dwheeler.com/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~x86-solaris"
```

Comment:
Please sort.

Reply:
```ebuild
KEYWORDS="~amd64 ~amd64-linux ~mips ~ppc ~ppc-macos ~sparc ~sparc-solaris ~x86 ~x86-linux ~x86-solaris"
```

Repoman still says these keywords are unsorted? I tried alphabetical order, maybe I missed something?

Reply:
```terminal
  KEYWORDS.unsorted             1
   dev-util/flawfinder/flawfinder-2.0.7.ebuild contains unsorted keywords
```

Reply:
please use ekeyword from the app-portage/gentoolkit